src/node_perf.cc

```diff
@@ -505,6 +506,7 @@ void ELDHistogram::DelayIntervalCallback(uv_timer_t* req) {
                  "mean", histogram->Mean());
   TRACE_COUNTER1(TRACING_CATEGORY_NODE2(perf, event_loop),
                  "stddev", histogram->Stddev());
+  histogram->EmitAfter(histogram->env(), histogram->get_async_id());
```

Comment:
The code in between here isn’t calling into JS or creating new async resourcces, is it? Calling async hooks isn’t cheap, so I would be careful about adding this in performance-relevant code…

Reply:
yeah, I was debating on whether to add this or not. I'll pull this back out.